Source/monster.cpp

```diff
@@ -3568,8 +3569,10 @@ void AddDoppelganger(Monster &monster)
 	}
 }
 
-void ApplyMonsterDamage(Monster &monster, int damage)
+void ApplyMonsterDamage(DamageType dmgType, Monster &monster, int damage)

```

Comment:
Any reason we shouldn't spell all these out (`damageType`)?

Reply:
Fixed